conftest/conftest.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: tekton.dev/v1alpha1
+kind: Task
+metadata:
+  name: conftest
+spec:
+  inputs:
+    resources:
+    - name: source
+      type: git
+      targetPath: source
+    params:
+    - name: files
+    - name: policy
+      default: "policy"
+    - name: output
+      default: "stdout"
+    - name: args
+      type: array
+      default: []
+
+  steps:
+  - name: conftest
+    workingdir: /workspace/source
+    image: instrumenta/conftest:latest
```

Comment:
@garethr I think we tend to prefer to have "fixed" version image instead of `latest`, does that make sense ?

Reply:
Makes sense. Making this configurable would be good in that case I presume. Do we have any conventions yet around parameter names doing the same?

Reply:
not really :thinking: It really depends on the images… on the go tasks we are using `version`, `buildah` uses a `stable` tags..